src/tests/test_dte_data_models.py

```diff
@@ -841,19 +921,27 @@ def test_init_fail_folio_ref_empty(self) -> None:
 
         obj = self.obj_2
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 folio_ref="",
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('folio_ref',),
-                    'msg': '("The length of \'folio_ref\' must be a value between 1 and 18", \'\')',
+                    'msg': (
+                        'Value error, ('
+                        '"The length of \'folio_ref\' must be a value between 1 and 18", \'\''
+                        ')'
+                    ),
                     'type': 'value_error',
-                }
+                },
```

Comment:
Revert change to reduce diff size (unless the comma was added by Black).

```suggestion
                }
```